hermes-test-helper/src/main/java/pl/allegro/tech/hermes/test/helper/endpoint/RemoteServiceEndpoint.java

```diff
@@ -107,7 +107,7 @@ public void setReturnedStatusCode(int statusCode) {
 
     public void waitUntilReceived(long seconds) {
         logger.info("Expecting to receive {} messages", expectedMessages.size());
-        await().atMost(adjust(new Duration(seconds, TimeUnit.SECONDS))).until(() -> receivedRequests.size() == expectedMessages.size());
+        await().atMost(adjust(new Duration(seconds, TimeUnit.SECONDS))).until(() -> receivedRequests.size() >= expectedMessages.size());
```

Comment:
maybe we should assert messages body instead of its number. Thanks this unexpected messages will not ruin our tests.